specs.json

```
@@ -557,7 +557,12 @@
   "https://www.w3.org/TR/css-contain-3/ delta",
   "https://www.w3.org/TR/css-content-3/",
   "https://www.w3.org/TR/css-counter-styles-3/",
-  "https://www.w3.org/TR/css-device-adapt-1/",
+  {
+    "url": "https://www.w3.org/TR/css-device-adapt-1/",
+    "nightly": {
+      "url": "https://drafts.csswg.org/css-viewport/"
```

The folder in the repo does not have a level, but the spec does, so suggest to be explicit about it.

```suggestion
      "url": "https://drafts.csswg.org/css-viewport-1/"
```